Dockerfile

```
@@ -495,6 +495,64 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+ARG GOROOT="/home/builder/go"
+ARG GOPATH="${GOROOT}/go"
+
+USER builder
+WORKDIR /home/builder
+COPY ./hashes/govmomi ./hashes
+RUN \
+  sdk-fetch hashes && \
+  tar xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz && \
+  mkdir -p ${GOPATH}/src/github.com/vmware && \
+  mv govmomi-${GOVMOMIVER} ${GOPATH}/src/github.com/vmware/govmomi && \
+  ln -s ${GOPATH}/src/github.com/vmware/govmomi
```

I think this can be shrunk down a bit as there's little `cd`-ing:

1. create directory by setting it as the `WORKDIR`
2. download the tarball into that directory (within `$GOPATH`)
3. unpack the tarball with first path component stripped off (ie: dropping the `govmomi-X.Y.Z/` leading path component)

```suggestion
WORKDIR /home/builder/go/src/github.com/vmware/govmomi
COPY ./hashes/govmomi ./hashes
RUN \
  sdk-fetch hashes && \
  tar --strip-components=1 -xf govmomi-${GOVMOMIVER}.tar.gz && \
  rm govmomi-${GOVMOMIVER}.tar.gz
```